bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/handler/LGWebOSTVSocket.java

```diff
@@ -187,7 +187,6 @@ public void connect() {
 
     public void disconnect() {
         Optional.ofNullable(this.session).ifPresent(s -> s.close());
-        this.session = null;
```

Comment:
Are you sure this is safe? Especially if the handler is re-initialized immediately (e.g. due to a configuration update)?

Reply:
I first added this line but the review comment was it was useless so I removed it.